cfnet/datasets.py

```diff
@@ -165,14 +159,39 @@ def __init__(self, data_configs: Dict):
         # prepare data
         self.prepare_data()
 
+    def name_to_config(self, data_name):
```

Comment:
move this function outside of the `TabularDataModule`.

E.g., write a function before the `TabularDataModule`.

```python
def _data_name2configs(data_name: str):
    ...
```